src/main/java/com/arias/kadai6/HelloController.java

```diff
@@ -0,0 +1,44 @@
+package com.arias.kadai6;
+
+
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloController {
+
+    @Async
+    @GetMapping("/hello")
+    public String exampleAsyncEndpoint() {
+
+        //クラスインスタンスの生成
+        RunWaiting1 runWaiting1 = new RunWaiting1();
```

Comment:
RunWaitingを依存性として注入してあげるようにすると、よりSpringの思想に沿った実装になるかと思います🙌

```suggestion
        RunWaiting1 runWaiting1 = new RunWaiting1();
        
            private final RunWaiting1 runWaiting1;
    private final RunWaiting2 runWaiting2;
    private final RunWaiting3 runWaiting3;

    public HelloController(RunWaiting1 runWaiting1, RunWaiting2 runWaiting2, RunWaiting3 runWaiting3) {
        this.runWaiting1 = runWaiting1;
        this.runWaiting2 = runWaiting2;
        this.runWaiting3 = runWaiting3;
    }
```